sdk/examples/nova.rs

```diff
@@ -0,0 +1,22 @@
+use nexus_sdk::{nova::seq::Nova, Prover, Verifiable};
+
+fn main() {
+    // expects for this program to be run from root of crate
+    // expects example programs (`nexus-zkvm/examples`) to have been built with `cargo build -r`
+    let pb = PathBuf::from(r"../target/riscv32i-unknown-none-elf/release/private_input");
```

Comment:
You can make it work from any dir as follows

https://github.com/nexus-xyz/nexus-zkvm/blob/3ab4d656b3f65ebb974888172438d38de8b83424/prover/examples/prove.rs#L22-L35